### PR TITLE
MRG, ENH: Add example of how to deal with EEG in MRI voxels

### LINKED ISSUE #3877: computing forward model using EEG electrode coords MRI voxel space?
hey, so i'm working through this tutorial here:
http://martinos.org/mne/stable/auto_tutorials/plot_forward.html#sphx-glr-auto-tutorials-plot-forward-py

about the head model and forward computation, but i'm having trouble getting my EEG electrodes in a form that MNE can recognize.
right now, i have electrode locations as freesurfer voxel coordinates (in the 256x256x256 freesurfer volume). 
electrode_scalp

i'm wondering if there is a way to bypass all the fiducial/digitization stuff, since i already have the electrode locations in MRI space (the data is from simultaneous EEG-FMRI experiments, so we were able to image the EEG cap in the scanner and consequently the electrodes are already aligned to the T1.

### PR DESCRIPTION
**Pull request by larsoner**
Closes #3877.

~~Example is a bit complicated because we have to fake the dataset using `sample`.~~

### CODE REVIEW

**Comment by larsoner** [comment]
https://15913-1301584-gh.circle-artifacts.com/0/dev/auto_tutorials/source-modeling/plot_eeg_mri_coords.html

**Comment by larsoner** [comment]
This one is also ready

File: tutorials/source-modeling/plot_eeg_mri_coords.py [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+img, affine = img.get_data().astype(float), img.affine
+# Mark them in the MRI with 5x5x5mm cubes
+for coord in np.round(np.concatenate([fid_vox, eeg_vox])).astype(int):
+    img[tuple(slice(c - 2, c + 2) for c in coord)] = 250
+# change "raw" to be more like what you'd acquire: no dig, no channel locations
+raw.load_data().pick_types(meg=False, eeg=True, exclude=(), stim=True)
+raw.info['dig'] = []
+for ch in raw.info['chs']:
+    ch['loc'].fill(0.)
+del head_to_vox, trans
```
Comment (agramfort) [inline tutorials/source-modeling/plot_eeg_mri_coords.py]: I am not super happy about exposing such complex / low level code. Can we have an alternative?

**Comment by larsoner** [comment]
@drammock feel free to merge if you're happy, leave comments, or push commits

**Comment by drammock** [comment]
I went ahead and merged without the CIs because my commit was a one-character change :)

**Pull request merged**